### PR TITLE
Remove  case review accepted for  form

### PR DESCRIPTION
**Pull request by bw2**

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "gzip-size": "4.1.0",
     "jquery": "3.2.1",
     "lodash": "4.17.4",
+    "npm": "^6.0.1",
```
Comment (hanars) [inline ui/package.json]: why is this added?

Comment (bw2) [reply ui/package.json]: hmm.. I ran `npm i npm` and it added itself. I'll take it out.

**Comment by bw2** [comment]
@anneodonnell requested this a couple days ago.
